patches/@typescript-eslint+eslint-plugin+8.32.1.patch

```diff
@@ -0,0 +1,59 @@
+diff --git a/node_modules/@typescript-eslint/eslint-plugin/dist/rules/no-deprecated.js b/node_modules/@typescript-eslint/eslint-plugin/dist/rules/no-deprecated.js
+index 80f8f64..d752160 100644
+--- a/node_modules/@typescript-eslint/eslint-plugin/dist/rules/no-deprecated.js
++++ b/node_modules/@typescript-eslint/eslint-plugin/dist/rules/no-deprecated.js
+@@ -153,12 +153,10 @@ exports.default = (0, util_1.createRule)({
+                     return false;
+             }
+         }
+-        function isInsideExportOrImport(node) {
++        function isInsideImport(node) {
+             let current = node;
+             while (true) {
```

Comment:
this infinite loop is probably the cause of broken CI ?

Reply:
Yes, it has to be, although could not say exactly why yet, it's ts-eslint implementation. I only made the function to return false when `ExportAllDeclaration` and `ExportNamedDeclaration` are met and made the function to guard only against imports 🤷🏻

Reply:
I completely removed that while loop in favour of 

```ts
+        function isInsideImport(node) {
+            const ancestors = context.sourceCode.getAncestors(node);
+            return ancestors.some(anc => anc.type.includes('Import') &&
+                anc.type !== utils_1.AST_NODE_TYPES.ImportExpression);
         }
```

pipeline still stucks 😞

Reply:
okay, now seems to be fixed, tests passing, pipeline line is not stucked anymore